core/editor/config-providers/editor-v2-config-provider.php

```diff
@@ -89,6 +90,8 @@ public function get_client_settings() {
 			'settings' => [
 				'urls' => [
 					'admin' => admin_url(),
+					// TODO: Replace with the final one once available.
```

Comment:
Ask for the real URL and do not merge it without it.

Reply:
Done